website/content/docs/internals/telemetry/metrics/merkle.mdx

```diff
@@ -0,0 +1,23 @@
+---
+layout: docs
+page_title: "Telemetry reference: Merkle tree metrics"
+description: >-
+  Technical reference for merkle tree telemetry values.
+---
+
+# Merkle tree telemetry
+
+Merkle tree telemetry provides information on the overall health of the
```

Comment:
Merkle trees are tightly tied to replication, maybe merge this with availability.mdx?

Reply:
That makes sense. The merkle tree metrics were called out in a separate section of the original list, so I wasn't 100% sure where they should go. I'm guessing the same applies to the WAL metrics?